logic/minecraft/auth/tasks/AuthenticateTask.cpp

```diff
@@ -14,20 +14,15 @@
  * limitations under the License.
  */
 
-#include <auth/flows/AuthenticateTask.h>
```

Comment:
The changes are either for adapting to changes in code elsewhere (amongst others not writing directly to member variables, yuck) or cleaning it up (adding braces around if, using ensureX). The changes will stay. According to your logic we shouldn't be doing wonko either.